### DISCUSSION TITLE
missing half of the parameters in MessageContent.MessagePhoto

### DISCUSSION THREAD

**Discussion by cherryymerryy** [egramtel/tdsharp#234] [Q&A]
![image](https://github.com/user-attachments/assets/6e83b638-aa5a-4fff-a45a-efbd5758dd3d)
![{8CCC81AA-AC08-4116-B831-51DB224B4193}](https://github.com/user-attachments/assets/fd44b8c7-2161-49b4-9f4b-c2eb85e5bcc8)

**Comment by ForNeVeR** [comment]
I am sorry, but to me the current binding looks correct: https://github.com/tdlib/td/blob/91aa6c9e4d0774eabf4f8d7f3aa51239032059a6/td/generate/scheme/td_api.tl#L415-L419

Not sure where all the other members are from, but not from the TL API.

**Reply by cherryymerryy** [reply]
Hmm, then how do I download photos?

**Reply by ForNeVeR** [reply]
Unfortunately, I have no idea. I am sorry. This is a question [for the upstream](https://github.com/tdlib/td) — we only provide bindings to the TD API that they generate.
